ui/imports/shared/views/chat/CompactMessageView.qml

```diff
@@ -51,6 +51,10 @@ Item {
 
     property string message: ""
 
+    property int prevMessageIndex
+    property string prevMsgTimestamp
```

Comment:
`ModelRole.PrevMsgTimestamp.int: "prevMsgTimestamp"`

some type discrepancy? :)

Reply:
`ModelRole.PrevMsgTimestamp.int` is just a model role enum mapping, the actual value exposed to qml using QVariant

Reply:
Correct but by assigning it to a `string` property you make it a string, not a number

Reply:
My point is, when you later want to format or display such timestamp in QML, you'd use sth like `new Date(prevMsgTimestamp).toLocaleString()` but that wouldn'T work as `Date` takes an `int` not a `string`

Reply:
Timestamps exposed to QML as QVariant, not int. Using string type for timestamp in QML is also strange for me. But if you want me to replace it here, imho i also should replace timestamps in the whole chat. @saledjenic @jrainville you were working on messaging part, what is yours opinion?

Reply:
It does make sense for the timestamp to be an int. I don't remember why it was put as a string at first, but I assume it was a mistake or just a misunderstanding.

Reply:
> Timestamps exposed to QML as QVariant, not int. Using string type for timestamp in QML is also strange for me. But if you want me to replace it here, imho i also should replace timestamps in the whole chat. @saledjenic @jrainville you were working on messaging part, what is yours opinion?

Well of course it's a `QVariant` but inside it holds an `int` not a `string`

Reply:
@MishkaRogachev if we're talking about `prevMsgTimestamp` it's coming from [here](https://github.com/status-im/status-desktop/blob/master/src/app/modules/shared_models/message_item.nim#L249-L278) and that's why it's string, cause entire object is passed as string to qml and parsed there.

Reply:
So for now i would prefer to use string and if we want to change it would be much better to create new issue for that

Reply:
OK